charts/ceph-csi-cephfs/templates/nodeplugin-daemonset.yaml

```diff
@@ -80,6 +80,9 @@ spec:
             - "--metricsport={{ .Values.nodeplugin.grpcMetrics.containerPort }}"
             - "--metricspath=/metrics"
             - "--enablegrpcmetrics=true"
+{{- end }}
+{{- if .Values.nodeplugin.forcecephkernelclient }}
+            - "--forcecephkernelclient=true"
```

Comment:
this should be `--forcecephkernelclient=Values.nodeplugin.forcecephkernelclient`